### PR TITLE
Update financial services page

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Update finacial services page following: https://docs.google.com/document/d/1SEd2nV0GplBTG4lZ9RuhjNXrphU22nGP4Fpgd5g_9BQ/edit#

## QA

- Go to https://ubuntu-com-9024.demos.haus/financial-services check against the docs

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3612

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @albertkol. Can we make a few changes please:

- Can each section have h2/h4 titles please

image

- The 451 Research logo is a bit small, can we try 32px high

image

- Can we add a bit more space above this title please, to match the "Software strategy acceleration" section above

image

- Can we also vertically centre both the Ubuntu and Canonical logos in their sections

Thanks

**Comment by albertkol** [comment]
Ready for review again @spencerbygraves

File: templates/financial-services/index.html [hunk 1]
```diff
@@ -230,175 +222,338 @@ <h3>Streamline business with containers for legacy apps</h3>
...
+        </ul>
       </div>
-      <div class="col-4 u-align--right u-vertically-center u-hide--small">
+    </div>
+    <hr />
+    <div class="p-strip is-shallow row u-equal-height u-no-padding--left u-no-padding--right">
+      <div class="col-8">
+        <h2>Increase developer productivity</h2>
+        <h4>Ubuntu workstations for engineering</h4>
+        <p>An <a href="/desktop">Ubuntu workstation</a> is the <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">professional developer’s preference</a>. Highly secure, reliable and with low cost of ownership, Ubuntu workstations are widely adopted by application engineering teams at financial institutions.</p>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:486]:
```suggestion
        <p>An <a href="/desktop">Ubuntu workstation</a> is the <a class="p-link--external" href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">professional developer’s preference</a>. Highly secure, reliable and with low cost of ownership, Ubuntu workstations are widely adopted by application engineering teams at financial institutions.</p>
```

File: templates/financial-services/index.html [hunk 2]
```diff
@@ -230,175 +222,338 @@ <h3>Streamline business with containers for legacy apps</h3>
...
+        <p>Canonical provides 24x7 support and <a href="https://landscape.canonical.com/">management tools</a> for monitoring, managing, patching and compliance reporting on all your Ubuntu workstations.</p>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:487]:
```suggestion
        <p>Canonical provides 24x7 support and <a class="p-link--external" href="https://landscape.canonical.com/">management tools</a> for monitoring, managing, patching and compliance reporting on all your Ubuntu workstations.</p>
```

File: templates/financial-services/index.html [hunk 3]
```diff
@@ -230,175 +222,338 @@ <h3>Streamline business with containers for legacy apps</h3>
...
+        <p>
+          <a href="https://buy.ubuntu.com/?_ga=2.170240736.1542483846.1604916464-1912709568.1604434903">Learn about Ubuntu Advantage&nbsp;&rsaquo;</a>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:489]:
```suggestion
          <a class="p-link--external" href="https://buy.ubuntu.com/?_ga=2.170240736.1542483846.1604916464-1912709568.1604434903">Learn about Ubuntu Advantage</a>
```

File: templates/financial-services/index.html [hunk 4]
```diff
@@ -230,175 +222,338 @@ <h3>Streamline business with containers for legacy apps</h3>
...
-    <div class="col-6 p-card">
-      <h3 class="p-heading-icon__title p-heading--four">Support</h3>
-      <p>
-        <a href="/support">24-hour support services with FIPS certification for compliance requirements&nbsp;&rsaquo;</a>
-      </p>
+    <div class="p-strip is-shallow row u-equal-height u-no-padding--left u-no-padding--right">
+      <div class="col-6">
+        <h2>Software strategy acceleration</h2>
+        <h3>Trusted by executives. Loved by leaders.</h3>
+        <p>The <a href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">best developers choose Ubuntu</a>.</p>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:514]:
```suggestion
        <p>The <a class="p-link--external" href="https://recruit-c7ff.kxcdn.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">best developers choose Ubuntu</a>.</p>
```

File: templates/financial-services/index.html [hunk 5]
```diff
@@ -83,108 +88,109 @@ <h2 class="p-muted-heading u-align--center">Ubuntu is used by</h2>
...
+              width="86",
+              height="32",
+              hi_def=True,
+              loading="auto|lazy",
+              attrs={"style": "margin-right: 1rem"}
               ) | safe
             }}
-            <h4 class="p-heading-icon__title">Whitepaper</h4>
+            <div class="p-heading-icon__title u-no-margin">
+              <a href="https://ubuntu.com/engage/finserv-multi-cloud">Find out why 60% of financial services turn to multi-cloud</a>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:112]:
```suggestion
              <a href="/engage/finserv-multi-cloud">Find out why 60% of financial services turn to multi-cloud</a>
```

File: templates/financial-services/index.html [hunk 6]
```diff
@@ -194,33 +200,19 @@ <h3>Getting started with AI</h3>
...
-            <h4 class="p-heading-icon__title">Whitepaper</h4>
-          </div>
-        </div>
-        <h3>Streamline business with containers for legacy apps</h3>
-        <p>Discover how Linux container technologies offer control and flexible provisioning while driving greater density, efficiency and manageability, without additional hardware or infrastructure investment. Different approaches work for next-generation apps and legacy apps. Ubuntu has them all.</p>
-        <p><a href="/blog/the-no-nonsense-way-to-accelerate-your-business-with-containers" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Download whitepaper', 'eventLabel' : 'Accelerate your business with containers', 'eventValue' : undefined });">Download whitepaper&nbsp;&rsaquo;</a></p>
+        <h2>Reduce IT operations cost</h2>
+        <h4>Model driven operators</h4>
+        <p>Canonical leads multi-cloud application management with the <a
+          href="https://juju.is/mission">universal operator pattern</a> to speed up operations and drive down the cost of integration.</p>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:206]:
```suggestion
          class="p-link--external" href="https://juju.is/mission">universal operator pattern</a> to speed up operations and drive down the cost of integration.</p>
```

File: templates/financial-services/index.html [hunk 7]
```diff
@@ -230,175 +222,338 @@ <h3>Streamline business with containers for legacy apps</h3>
...
+      </div>
+    </div>
+    <hr />
+    <div class="p-strip is-shallow row u-equal-height u-no-padding--left u-no-padding--right">
+      <div class="col-8">
+        <h2>Enhance business resilience</h2>
+        <h4>Data centre of the future</h4>
+        <p>Future-proof your data centres for hybrid cloud and cloud-native applications.
+          <a href="/server">Ubuntu server</a> brings economic and technical scalability to your data centre. Turn your data centre into a bare metal cloud with
+          <a href="https://maas.io/">MAAS</a>, an automated and optimised provisioning system for your production hardware.</p>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:432]:
```suggestion
          <a class="p-link--external" href="https://maas.io/">MAAS</a>, an automated and optimised provisioning system for your production hardware.</p>
```

File: templates/financial-services/index.html [hunk 8]
```diff
@@ -230,175 +222,338 @@ <h3>Streamline business with containers for legacy apps</h3>
...
-      <div><a href="/financial-services/contact-us" class="p-button--positive js-invoke-modal">Contact us</a></div>
+    <div class="u-fixed-width">
+      <hr class="p-separator">
     </div>
-    <div class="col-4 u-hide--small u-align--center">
-      {{
+    <div class="p-strip is-shallow row u-equal-height u-no-padding--left u-no-padding--right">
+      <div class="col-6">
+        <h2>Innovate with Canonical</h2>
+        <h3>Trusted by executives. Loved by leaders.</h3>
```
Comment (bethcollins92) [inline templates/financial-services/index.html:537]: Remove this line 
```suggestion
        
```

**Review by bethcollins92 — APPROVED** [review]
Looking good! Just a few small changes 👍🏼

File: templates/financial-services/index.html [hunk 9]
```diff
@@ -2,78 +2,83 @@
...
+        {{ image (
+          url="https://assets.ubuntu.com/v1/b41ec3f5-SBI-Bits_logo.png",
+          alt="SBI Bits",
+          width="160",
+          height="49",
+          hi_def=True,
+          loading="auto|lazy"
           ) | safe
         }}
       </li>
```
[the remark this answers is not in the document]
Comment (bethcollins92) [reply templates/financial-services/index.html:84]: ahhh yes, apologies missed that 👍🏼

File: templates/financial-services/index.html (hunk 1 above)
Comment (albertkol) [reply templates/financial-services/index.html:486]: Done.

File: templates/financial-services/index.html (hunk 2 above)
Comment (albertkol) [reply templates/financial-services/index.html:487]: Done.

File: templates/financial-services/index.html (hunk 3 above)
Comment (albertkol) [reply templates/financial-services/index.html:489]: Done.

File: templates/financial-services/index.html (hunk 4 above)
Comment (albertkol) [reply templates/financial-services/index.html:514]: Done.

File: templates/financial-services/index.html (hunk 5 above)
Comment (albertkol) [reply templates/financial-services/index.html:112]: Done.

File: templates/financial-services/index.html (hunk 6 above)
Comment (albertkol) [reply templates/financial-services/index.html:206]: Done.

File: templates/financial-services/index.html (hunk 7 above)
Comment (albertkol) [reply templates/financial-services/index.html:432]: Done.

File: templates/financial-services/index.html (hunk 8 above)
Comment (albertkol) [reply templates/financial-services/index.html:537]: Done.

**Pull request merged**